lib/mempool/mempool.js

```diff
@@ -953,6 +967,72 @@ class Mempool extends EventEmitter {
         0);
     }
 
+    // If we reject RBF transactions altogether we can skip these checks,
+    // because incoming conflicts are already rejected as double spends.
+    let conflicts = [];
+    if (this.options.replaceByFee)
+      conflicts = this.getConflicts(tx);
+
+    if (conflicts.length) {
```

Comment:
I feel like this `if` statement is boilerplate code in the sense where it just adds additional line that a reviewer needs to read and moves everything in its block two spaces over when the for loop below (`for (const conflict of conflicts)`) would just never run if `conflicts.length` is `0`

Reply:
Hm thanks. The `if` does cut off a lot of unnecessary checks however, not just the `for` loop. Especially the `replacement-adds-unconfirmed` check which iterates through all a TX's inputs. I suppose I could move all that stuff out of `verify()` and into a new function `checkConflicts()` or something? Waddaya think?

Reply:
> The if does cut off a lot of unnecessary checks however

Good looks. I was trying to figure out how to move some code around so that `verify` no longer had a return value (like before this PR), but was worried that it would cause some performance issues around double checking things

Reply:
So, you think it's ok as is?

Reply:
so you think its ok as is?